st2common/st2common/content/bootstrap.py

```diff
@@ -329,14 +330,22 @@ def register_content():
     register_all = cfg.CONF.register.all
 
     if register_all:
-        register_triggers()
-        register_sensors()
-        register_runners()
-        register_actions()
-        register_rules()
-        register_aliases()
-        register_policies()
-        register_configs()
+        with Timer(key='st2.register.triggers'):
```

Comment:
Looks like we only time those operations when ``register_all` is used and not when using a specific flag to only register a subset of resources.

I think it would be better to move this instrumentation inside ``register_<resource>`` functions.

This way it should work "out of the box" for all the approaches (using `--register-all` or ``--register-actions --register-rules``, etc.).

Reply:
Good point. I'll fix this.